Alignment/MillePedeAlignmentAlgorithm/scripts/mps_alisetup.py

```diff
@@ -282,14 +282,26 @@ def _create_mille_jobs(self):
             tmpFile = re.sub(collection_regex,
                              'setupCollection = \"'+dataset["collection"]+'\"',
                              tmpFile)
-            if dataset['cosmicsZeroTesla']:
-                tmpFile = re.sub(czt_regex,
-                                 'setupCosmicsZeroTesla = True',
-                                 tmpFile)
-            if dataset['cosmicsDecoMode']:
-                tmpFile = re.sub(cdm_regex,
-                                 'setupCosmicsDecoMode = True',
-                                 tmpFile)
+            if dataset["collection"] in ("ALCARECOTkAlCosmicsCTF0T",
```

Comment:
@adewit as we have other track collections in the ALCARECO files, and we might add more in view of the work on CDCs I am wondering if this could become a check against a generic `ALCARECOTkAlCosmics*`.

Reply:
I think so - naively would expect I can turn the if statement around... will try a bit later